nym-vpn-core/crates/nym-vpn-lib/src/tunnel_state_machine/tunnel_monitor.rs

```diff
@@ -753,6 +781,14 @@ impl TunnelMonitor {
                         }
                     }
                 }
+                close_event = bridge_close_rx.recv() => {
+                    if close_event.is_some() {
+                        tracing::info!("Bridge close signal received. Exiting");
+                    } else {
+                        tracing::info!("Bridge channel is closed. Exiting");
```

Comment:
That doesn't fly. If the bridge isn't enabled, `bridge_close_tx` is dropped, so this branch is hit, preventing all connection attempts

Reply:
Fixed